enterprise/server/remote_execution/filecache/filecache.go

```diff
@@ -103,14 +104,14 @@ func evictFn(v *entry, reason lru.EvictionReason) {
 // NewFileCache constructs an fileCache with maxSize that will cache files
```

Comment:
Can the filecache changes can be reverted now?

Reply:
No, the file cache needs to pass the directory to `uuid.GetHostID`, since the directory has to be stored in the environment since it is coming from a flag in `enterprise`.